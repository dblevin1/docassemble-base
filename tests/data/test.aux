\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {section}{\numberline {I.}Section the First}{1}{section.1}}
\newlabel{section-the-first}{{I.}{1}{Section the First}{section.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {A.}Subsection the smooch}{1}{subsection.1.1}}
\newlabel{subsection-the-smooch}{{A.}{1}{Subsection the smooch}{subsection.1.1}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.}The dumber subbysubbysection}{2}{subsubsection.1.1.1}}
\newlabel{the-dumber-subbysubbysection}{{1.}{2}{The dumber subbysubbysection}{subsubsection.1.1.1}{}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {a.}asdfjas dajsdfoa sjfasdoijfasd oiasdf}{2}{paragraph.1.1.1.1}}
\newlabel{asdfjas-dajsdfoa-sjfasdoijfasd-oiasdf}{{a.}{2}{asdfjas dajsdfoa sjfasdoijfasd oiasdf}{paragraph.1.1.1.1}{}}
\@writefile{toc}{\contentsline {subparagraph}{\numberline {i.}arwerwerwerwe rwerwe rwerw erw}{2}{subparagraph.1.1.1.1.1}}
\newlabel{arwerwerwerwe-rwerwe-rwerw-erw}{{i.}{2}{arwerwerwerwe rwerwe rwerw erw}{subparagraph.1.1.1.1.1}{}}
