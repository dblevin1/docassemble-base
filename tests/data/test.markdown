---
fontsize: 12pt
fontfamily: mathptmx
papersize: letterpaper
documentclass: article
numbersections: true
geometry: left=1in,right=1in,top=1in,bottom=1in
header-includes:
  - \usepackage{setspace}
  - \usepackage{fancyhdr}
  - \usepackage{indentfirst}
  - \renewcommand{\thispagestyle}[1]{}
  - \renewcommand{\headrulewidth}{0pt}
  - \renewcommand{\thesection}{\Roman{section}.} 
  - \renewcommand{\thesubsection}{\Alph{subsection}.}
  - \renewcommand{\thesubsubsection}{\arabic{subsubsection}.}
  - \renewcommand{\theparagraph}{\alph{paragraph}.}
  - \renewcommand{\thesubparagraph}{\roman{subparagraph}.}
  - \usepackage{titlesec}
  - \titleformat{\section}{\bfseries}{\thesection}{1em}{}
  - \titleformat{\subsection}{\bfseries}{\thesubsection}{1em}{}
  - \titleformat{\subsubsection}{\bfseries}{\thesubsubsection}{1em}{}
  - \titleformat{\paragraph}{\bfseries}{\theparagraph}{1em}{}
  - \titleformat{\subparagraph}{\bfseries}{\thesubparagraph}{1em}{}
  - \titlespacing*{\section}{0pt}{12pt}{12pt}
  - \titlespacing*{\subsection}{3em}{12pt}{12pt}
  - \titlespacing*{\subsubsection}{6em}{12pt}{12pt}
  - \titlespacing*{\paragraph}{9em}{12pt}{12pt}
  - \titlespacing*{\subparagraph}{12em}{12pt}{12pt}
  - \newcommand{\myfontfamily}{\rmfamily}
  - \newcommand{\myfontsize}{12pt}
  - \newcommand{\verytopskip}{0in}
  - \newcommand{\myshow}[1]{#1}
  - \usepackage{enumitem}
  - \newenvironment{mytitle}{\bfseries\centering }{\par}
  - \pagestyle{fancy}
  - \fancyhead{}
  - \fancyfoot{}
  - \fancyfoot[CE,CO]{\myfontfamily -~\thepage~-}
  - \setlength{\parindent}{0.5in}
  - \setlength{\parskip}{0pt}
  - \setlist[enumerate]{leftmargin=0pt,itemindent=1in,align=left,labelwidth=0.5in,labelsep=0in,topsep=0pt,partopsep=0pt,parsep=0pt,itemsep=0pt}
  - \setlist[trivlist]{topsep=0pt,partopsep=0pt}
include-before:
  - \raggedright
  - \myfontfamily
...
\vspace*{\verytopskip}
\begin{center}
\uppercase{\textbf{In the Court of Common Pleas of Philadelphia County}}

\vspace*{\myfontsize}

\noindent\begin{tabular}{@{}p{0.49\textwidth}|@{\hspace{1em}}p{0.49\textwidth}@{}}
{asdfasdf
\newline
asdfasdf
\newline
\newline
sdfasdf
\newline
asdfasdf} & {werwer
\newline
werwe
\newline
\newline
werwerwer
\newline
ewrewre} \\ 
\end{tabular}

\vspace*{\myfontsize}

\uppercase{\textbf{Complaint for Custody}}

\end{center}

\doublespacing

For the following reasons, I should get the relief I want:

#. I am [not a child](http://google.com).
#. I am a very important person.
#. You are not a very important person, despite the fact that you like to take important people out for dinner at the fanciest restaurants in Detroit.  You should be ashamed of yourself for being such a noob.

My contact information is:

| Jonathan Pyle
| 113 Gladstone Road
| Lansdowne, PA 19050

Isn't it grand?

# Section the First #

Comfort reached gay perhaps chamber his six detract besides add. Moonlight newspaper up he it enjoyment agreeable depending. Timed voice share led his widen noisy young. On weddings believed laughing although material do exercise of. Up attempt offered ye civilly so sitting to. She new course get living within elinor joy. She her rapturous suffering concealed. 

Unpleasant astonished an diminution up partiality. Noisy an their of meant. Death means up civil do an offer wound of. Called square an in afraid direct. Resolution diminution conviction so mr at unpleasing simplicity no. No it as breakfast up conveying earnestly immediate principle. Him son disposed produced humoured overcame she bachelor improved. Studied however out wishing but inhabit fortune windows. 

## Subsection the smooch ##

Day handsome addition horrible sensible goodness two contempt. Evening for married his account removal. Estimable me disposing of be moonlight cordially curiosity. Delay rapid joy share allow age manor six. Went why far saw many knew. Exquisite excellent son gentleman acuteness her. Do is voice total power mr ye might round still. 

Picture removal detract earnest is by. Esteems met joy attempt way clothes yet demesne tedious. Replying an marianne do it an entrance advanced. Two dare say play when hold. Required bringing me material stanhill jointure is as he. Mutual indeed yet her living result matter him bed

### The dumber subbysubbysection ###

Their could can widen ten she any. As so we smart those money in. Am wrote up whole so tears sense oh. Absolute required of reserved in offering no. How sense found our those gay again taken the. Had mrs outweigh desirous sex overcame. Improved property reserved disposal do offering me. 

#### asdfjas dajsdfoa sjfasdoijfasd oiasdf ####

Building mr concerns servants in he outlived am breeding. He so lain good miss when sell some at if. Told hand so an rich gave next. How doubt yet again see son smart. While mirth large of on front. Ye he greater related adapted proceed entered an. Through it examine express promise no. Past add size game cold girl off how old. 

Examine she brother prudent add day ham. Far stairs now coming bed oppose hunted become his. You zealously departure had procuring suspicion. Books whose front would purse if be do decay. Quitting you way formerly disposed perceive ladyship are. Common turned boy direct and yet. 

##### arwerwerwerwe rwerwe rwerw erw #####

I would like you to pay me $5.00 which represents 5% interest on the money you owe me.  Now!
