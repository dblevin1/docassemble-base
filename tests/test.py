#! /usr/bin/python
from docassemble import Interview

interview = Interview("tests/testinterview/questions.yml")
interview.assemble()
