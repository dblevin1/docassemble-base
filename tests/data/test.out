\BOOKMARK [1][-]{section.1}{\376\377\000S\000e\000c\000t\000i\000o\000n\000\040\000t\000h\000e\000\040\000F\000i\000r\000s\000t}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{\376\377\000S\000u\000b\000s\000e\000c\000t\000i\000o\000n\000\040\000t\000h\000e\000\040\000s\000m\000o\000o\000c\000h}{section.1}% 2
\BOOKMARK [3][-]{subsubsection.1.1.1}{\376\377\000T\000h\000e\000\040\000d\000u\000m\000b\000e\000r\000\040\000s\000u\000b\000b\000y\000s\000u\000b\000b\000y\000s\000e\000c\000t\000i\000o\000n}{subsection.1.1}% 3
